Tiltfile

```diff
@@ -253,7 +254,6 @@ def create_identity_secret():
 
     os.putenv("AZURE_CLIENT_SECRET_B64", base64_encode(os.environ.get("AZURE_CLIENT_SECRET")))
     local("cat templates/azure-cluster-identity/secret.yaml | " + envsubst_cmd + " | " + kubectl_cmd + " apply -f -", quiet = True, echo_off = True)
-    os.unsetenv("AZURE_CLIENT_SECRET_B64")
```

Comment:
why is this change needed?

Reply:
This was necessary to make sure that variable is still set since it was referenced by the ASO global settings Secret for fallback credentials. Now that those are all blank though I think this change can be reverted.